.gitignore

```
@@ -273,14 +273,14 @@ etlman/media/
 
 .pytest_cache/
 .ipython/
-# generated by gulp from project.scss
+
+# generated by webpack from index.js
+etlman/static/js/*
```

I'm getting a few more auto-generated files that would be nice to ignore:
```suggestion
etlman/static/js/*
etlman/static/*.worker.js
etlman/static/*.ttf
```